1907-03-11.xml

```diff
@@ -0,0 +1,1824 @@
+                        <p>C. H. SCHOELLER, Agent In Alexandria, Cleopatra Lane.</p>
+                        <p>Messrs. THOS. COOK &amp; SON (Egypt) LTD., and CARL STANGENS REISEBUREAN are
+                            anthorised to sell tickets in CAIRO and ALEXANDRIA, <measure type="indexNo">31-8-905</measure></p> 
+                    </div>
+                    <div type="advert" xml:id="deg-ad-als03">
+                        <head>Austrian Lloyd’s Steam Navigation</head>
+                        <p><hi rend="bold">Alexandria-Brindisi-Venice Trieste.</hi> Express Mail Service.
+                            Steamers leave Alexandria Saturdays 4 p.m. arrive at Brindisi Tuesdays 5 a.m. in
+                            time for Express to Milan, Lucerne, Paris, Vienna, Berlin, London, leaving
+                            Brindisi 7 a.m. Arrival at Venice Wednesday about 9.30 a.m. Passengers may
+                            proceed from Venice to Switzerland, Paris and London by the 2 p.m. and 11.5 p.m.
+                            – Expresses. Arrival at Trieste Wednesday about 3 p.m. in connection with the
+                            Train de Luxe proceeding the same evening to Vienna, Frankfort, Cologne,
+                            Brussel, Ostende and London. Arrival at London every Friday 5.12 p.m.</p>
+                        <p>Special Train Service from Cairo Station to Alexandria Quay alongside steamer,
+                            Saturdays at 9.30 a.m.</p>
+                        <p><hi rend="bold">Intermediate Service: Alexandria-Brindisi &amp; Trieste
+                            Line.</hi></p>
+                        <p>Steamer leaves Alexandria every Thursday 10 a.m. On and from <date when="1907-04-11">April 11th</date> at 4 p.m.</p>
+                        <p><hi rend="bold">Syrian-Caramanian Line.</hi> Port Said, Jaffa, Beyrout, Tripoli,
+                            Alexandretta, Mersina, <date when="1907-02-18">Febr. 18</date>; <date when="1907-03-04">March 4</date>, <date when="1907-03-18">18</date> ; <date when="1907-04-01">April 1</date>. <date when="1907-04-15">15</date>, <date when="1904-04-29">29</date> ; <date when="1907-05-13">May 13</date>, <date when="1905-05-27">27</date> ; <date when="1907-06-10">June 10</date>, <date when="1907-06-24">24</date>.</p>
+                        <p><hi rend="bold">Syrian-Cyprus Line.</hi> Beyrout, Limassol, Larnaca, Mersina,
+                            <date when="1907-02-26">Febr. 26</date>; <date when="1907-03-12">March
+                                12</date>, <date when="1907-03-26">26</date> ; <date when="1907-04-09">April
+                                    9</date>. <date when="1907-04-23">23</date> ; <date when="1907-05-07">May
+                                        7</date>, <date when="1905-05-21">21</date> ; <date when="1907-06-04">June
+                                            4</date>, <date when="1907-06-18">18</date>.</p>
+                        <p><hi rend="
```

Comment:
Here you close the page division too early, thus the rest of this page no longer belongs to page 2. Please check your outline window and fix this.

Reply:
It seems that my comment is not shown in the accurate line. Please check you line 1697 and your outline window to see.